gcal/gcal.go

```diff
@@ -61,7 +62,7 @@ func (g *GCal) Schedule(sched *schedule.Schedule) error {
 	}
 
 	for i, ie := range internalEvents {
-		_, err := g.svc.Events.Insert(g.CalendarID, ie.GcalEvent).SendUpdates("none").Do()
+		_, err := g.svc.Events.Insert(g.CalendarID, ie.GcalEvent).SendUpdates("all").Do()
```

Comment:
Doesn't this mean every time the action runs I'll get a whole bunch of emails, since you're clearing and re-creating the events every time?

Reply:
Yes, I think it will. We can opt to disable notifications while still having it on your calendar.

Reply:
Changed it to `externalOnly` to reduce email spam, though I think everybody on the rotation currently uses Google Calendar.